src/radiation/radiation.cpp

```diff
@@ -130,17 +118,23 @@ void ablate::radiation::Radiation::Setup(const solver::Range& cellRange, ablate:
                 virtualcoord[ipart].y = centroid[1] + (virtualcoord[ipart].ydir * 0.1 * minCellRadius);
                 virtualcoord[ipart].z = centroid[2] + (virtualcoord[ipart].zdir * 0.1 * minCellRadius);
 
+                // Init hhere to default value
```

Comment:
@owenmylotte I don't think we had a default value here, is this ok?

Reply:
That looks good to me. The path length should always be set before it's entered in the vector so having it uninitialized shouldn't cause problems. Zero is good. Was this causing a bug?

Reply:
I'm not sure, either this or FaceIntersect was reporting using uninitialized data.